lampod/src/ln/peer_manager.rs

```diff
@@ -132,10 +132,15 @@ impl LampoPeerManager {
             .clone()
             .ok_or(error::anyhow!("channel manager is None"))?;
         let alias = self.conf.alias.clone().unwrap_or_default();
+        let addr = self
+            .conf
+            .announce_addr
+            .clone()
+            .unwrap_or_else(|| "0.0.0.0".to_string());
```

Comment:
```suggestion
            .unwrap_or_else(|| "127.0.0.1".to_string());
```

Better do not expose to any IP, this is a privacy leack

Reply:
Oops